subworkflows/local/db_check.nf

```diff
@@ -17,10 +18,22 @@ workflow DB_CHECK {
         .dump(tag: "db_split_csv_out")
         .map { create_db_channels(it) }
         .dump(tag: "db_channel_prepped")
-        .set{ dbs }
+
+    parsed_samplesheet
+        .branch {
+            untar: it[1].toString().endsWith(".tar.gz")
+            skip: true
+        }
+        .set{ ch_dbs_for_untar }
```

Comment:
I personally like channel assignment better than using `.set` because then I can quickly scan where channels are created. I know that this style is quite widely used, though.

Reply:
I actually flip flop either way, depending on the day it seems. But no strong opinion.

so I don't mind, feel free to make a suggestion comment to direct it and I can commit that in!